### DISCUSSION TITLE
Custom icon for folder notes

### DISCUSSION THREAD

**Discussion by p2kmgcl** [LostPaul/obsidian-folder-notes#231] [Ideas]
Since we are already using a real file as a folder note, it would be nice to read some properties from the file to customize how the folder note looks (thinking about Notion's UI).

My proposal is to read this information from the folder note:

```
---
icon: 🗻
---
```

and use it as icon on the file tree.

Do you think this is a good idea?
Would you accept a PR with this feature?

**Comment by TheFlatline** [comment]
This is actually the only feature I am missing from the old aidenx folder notes :)
(for reference the "icon:" frontmatter field is used with obsidian-iconize)

**Comment by hilsonp** [comment]
@p2kmgcl : Would you be so kind to share any solution/workaround/fork you have ?
This is indeed the last thing that is missing (A LOT) from the aidenx folder notes :-(
